### ISSUE TITLE
open Hapi server by fhir-face

### ISSUE THREAD

**Issue by coderalilty** [Ivana-/fhir-face#2]
Hi I try open my own hapi server by fhir-face but I got the next error

> Error: Failed to fetch http://localhost:8089/fhir/resource/Entity
console log:

> GET http://localhost:8089/fhir/Entity?_count=1000&.type=resource&_sort=.id 404 (Not Found)

**Comment by niquola** [comment]
Use aidbox :)

**Comment by Ivana-** [comment]
Yes, different fhir servers have some differencies in queries for metadata. Your example is for aidbox one, but you can see how choose hapi in the code. At least, current demo page (by the link on readme) compiled from current version of code, and it works by default with shared hapi server. Maybe I'l add a ui for this choice later
